src/components/ebay-lightbox-dialog/README.md

```diff
@@ -0,0 +1,23 @@
+# ebay-lightbox-dialog
+
+```marko
+<ebay-lightbox-dialog open a11y-close-text="Close Dialog">
+  <h1>Hello World</h1>
+</ebay-lightbox-dialog>
+```
+
+## Attributes
+
+Name | Type | Stateful | Required | Description
+--- | --- | --- | --- | ---
+`open` | Boolean | Yes | No | Whether dialog is open.
+`focus` | String | No | No | An id for an element which will receive focus when the dialog opens (defaults to close button).
+`mode` | String | No | No | Either "mini" or "default". Changes the dialog to be normal dialog or mini version (for use in infotip)
```

Comment:
Not sure how I feel about exposing `mode="mini"` seems like that should be exclusive to the infotip implementation.

Reply:
I exposed it because we will add different modes. Since mini was so tied to lightbox at least I have it here. 
I can hide the mode mini, but at least it gives developers a glimpse into what is being done.

Reply:
We also have confirm and alert variants to think about. These are very specialized types of lightbox. With just a header, description text, and one or two buttons. Given that they are not open to much customization, they could potentially be their own standalone components, e.g. ebay-confirm-dialog and ebay-alert-dialog, that wrap ebay-lightbox-dialog.

Reply:
Happy for you to leave alert and confirm for a follow up PR, but have a think about it and see how it might affect your thinking (if at all) for this current PR.